### ISSUE TITLE
Can't use some key anymore on azerty keyboard

### ISSUE THREAD

**Issue by cyrinux** [hyprwm/Hyprland#1351]
Hello 😁,

On a azerty keyboard, or maybe other mapping, I can use, and use those key without SHIFT.
So the warning is false, and block the usage of those key.

I can still use keycode, but should we keep this piece of code? Do you plan to remove it at a moment and it was just to alert first users ?
Can we maybe add a boolean in config to bypass this warning ?

https://github.com/hyprwm/Hyprland/blob/main/src/config/ConfigManager.cpp#L714

src/config/ConfigManager.cpp

```
    if (KEY == "exclam" || KEY == "asciicircum" || KEY == "at") { // just some
        parseError = "Your config contains (probably) wrong keys. The SHIFT keysym behavior has changed after v0.10.3beta. Please consult the wiki (Advanced configuring -> binds)";
        return;
    }

```

Regards

**Comment by vaxerski** [comment]
it was meant for when there was shifting. I forgot to remove this. Sorry

**Comment by vaxerski** [comment]
fixed in a572321f611d256f746366fbf16d36ed6a7de3e6
